.github/workflows/cypress.yaml

```diff
@@ -174,7 +174,7 @@ jobs:
                   record: true
                   parallel: true
                   command-prefix: "yarn percy exec --parallel --"
-                  config: '{"reporter":"cypress-multi-reporters", "reporterOptions": { "configFile": "cypress-ci-reporter-config.json" } }'
+                  config: '{"reporter":"cypress-multi-reporters", "reporterOptions": { "configFile": "cypress-ci-reporter-config.json" }, "morgan": false }'
```

Comment:
I don't know why the config in `cypress.config.ts` isn't working here, and I don't know if this is going to fix it.

Reply:
Narrator: it did not fix it. The test logs include:

```
[percy] Running "cypress run --headed --record --parallel --config {"reporter":"cypress-multi-reporters", "reporterOptions": { "configFile": "cypress-ci-reporter-config.json" }, "morgan": false } --ci-build-id sha-c5befa1ef8ed72f0d09ae4fb5df21b4536b324f7-time-1689141941"
The following configuration option is invalid:

 - morgan
 ```